### ISSUE TITLE
Error strings doesn't appear if characterRestriction is exceeded

### ISSUE THREAD

**Issue by Assem-Hafez** [n4kz/react-native-material-textfield#15]
When i set the value of the error, the error message desapears if characterRestriction is exceeded.

Note that if the title and error are set it behaves normally and the error message appear even if the characterRestriction is exceeded.

**Comment by n4kz** [comment]
Failed to reproduce on RN 44.0 in example project, could you provide more details and code sample?

**Comment by Assem-Hafez** [comment]
Sorry for not being specific.
I found that the problem happens in a very specific scenario not as general as the issue was submitted.
if you tried the code below the error message doesn't show.
```
import React from 'react';
import { TextField } from 'react-native-material-textfield';

var App = React.createClass({
    getInitialState() {
        return {}
    },
    render: function () {
        var label = "label";
        var maxlength = 7;
        var description = "";
        return (
            <TextField
                label={label}
                value={this.state.viewValue}
                onChangeText={this.onChange}
                characterRestriction={maxlength}
                title={description}
                error={this.state.errorMessage} />
        );
    },
    onChange: function (value) {
        let maxlength = 7;
        this.setState({ viewValue: value, errorMessage: value.length > maxlength ? "error" : undefined })
    }
```
if you changed the description to contain a string, for example a space `description=" "`, the message will show normally.
also if the error message condition written in the `onChange` function was changed to be `errorMessage: value.length >= maxlength` (adding an equal) the message will show normally.

wish this description helps.

**Comment by n4kz** [comment]
Thanks for detailed description. Looks like it's strange rendering bug, `Helper` component was re-rendered on error, but remained invisible. Fixed by passing text as `children`. Released in `0.5.3`.
